README.md

```diff
@@ -111,3 +116,19 @@ note ={PMID: 33155465},
 URL = {https://doi.org/10.1021/acs.jcim.0c00839}
 }
 ```
+## List of publications
+- https://pubs.acs.org/doi/full/10.1021/acs.jcim.1c00608 
+- https://www.sciencedirect.com/science/article/pii/S2001037023003495
+
+- https://doi.org/10.1021/acs.jmedchem.1c01803 ?
+- https://10.1021/acs.jmedchem.2c00931 ?
```

Comment:
This link doesn't work for me (at the moment)?

Reply:
Added correct link to the end of the list

Reply:
I just checked it - I would keep this paper in the list :)